.env

```diff
@@ -0,0 +1,4 @@
+API_PORT=":1323"
+REDIS_PORT=":6379"
+AWS_ACCESS_KEY_ID=[---ENTER AWS ACCESS KEY ID HERE ---]
+AWS_SECRET_ACCESS_KEY=[---ENTER AWS SECRET ACCESS KEY HERE ---]
```

Comment:
@tmilewski is this file even supposed to be committed at all?

I imagine this would be a file we would keep in our `.gitignore`, to prevent any accidental  leaking of our access keys + secret!

Also not including this file would be good, because then if a user forgot to add this we would immediately be notified @ main.go line 41.